tests/integration/test_k8s_with_cluster.py

```diff
@@ -446,11 +454,45 @@ def test_cli_cronjob_handler_crud():
                 "delete",
                 "cronjobs",
                 "bodywork-test-project",
-                f"--namespace={BODYWORK_DEPLOYMENT_JOBS_NAMESPACE}",
+                f"--namespace={BODYWORK_NAMESPACE}",
             ]
         )
 
 
+def test_deployment_with_ssh_github_connectivity_from_file(
```

Comment:
Why do we need this, as well as `test_deployment_with_ssh_github_connectivity`?

Reply:
We're thinking of testing this as the previous test is using the environment variable (as you can see from the fixture is used).

Reply:
I've had chance to think about this a bit more and I think we can get rid of `test_deployment_with_ssh_github_connectivity` instead. This is because `test_deployment_with_ssh_github_connectivity_from_file` will test the SSH with the environment variable as well, when the Bodywork job runs in the container.

Reply:
Cool.